package.json

```diff
@@ -15,7 +15,20 @@
     "biojs"
   ],
   "main": "build/pileup.js",
-  "browser": "build/pileup.js",
+  "browser": "src/pileup.js",
+  "browserify": {
```

Comment:
They depend on this [wzrd.in](http://wzrd.in) service for browserification:

![screen shot 2015-06-11 at 4 34 24 pm](https://cloud.githubusercontent.com/assets/7809/8117650/c1d8821c-1057-11e5-805d-adf1bf2a2cb2.png)

This problem manifested itself on BioJS, but I think this flexibility might become handy for other users who depend on the *node-browserify* tool for browserification.

Sharing settings would be great, let me investigate that option. If it is not possible, then I will go with option 1.